### PR TITLE
Ru/mobile updates

### PR DESCRIPTION
**Pull request by rudouglas**
# Summary

This update adds new components and updates both the Mobile and Mobile Crash Analytics Quickstarts:

## Mobile Crash Analytics
 - Adds Alert conditions
 - Adds Mobile Alert documentation
 - Changes the Icon

## Mobile
 - Adds Alert conditions
 - Adds Mobile Alert documentation
 - Updates the existing Dashboard to use variables and changes the layout

### CODE REVIEW

**Comment by aswanson-nr** [comment]
Well, hello there 👋 
How did you immediately break my bot responses? haha
Good to see you @rudouglas !

**Comment by rudouglas** [comment]
Breaking things is what i do best @aswanson-nr you know that 😛 good to see you too 

Are those things that I need to revise?

**Review by brammerl — DISMISSED** [review]

File: dashboards/mobile/mobile.json [hunk 1]
```diff
@@ -19,11 +19,11 @@
...
...this text box.\n\nYou may notice after installation that charts are showing no data. You can select a specific app to be the default in the variable dropdown labelled `Select Mobile App`:\n- [ ] Click on the ✏️ icon i nthe top right-hand corner of the dashboard\n- [ ] Click on the `Select Mobile App` variable dropdown in the top left-hand corner and then click **Edit**\n- [ ] In the settings sidebar select an app from the `Default value` dropdown\n\nThe dashboard will default to data from this app and you can select any other app from the `Select Mobile App` dropdown.\n\n > NOTE: While some of the charts in this dashboard might work out of the box without an app selected, please bear in mind the results displayed will be an aggreagation of *all* mobile apps in your account.\n \n ---\n \n ⚠️ The dropdown list of apps is currently limited to 10000. If you have more than 10000 apps in your account you can adjust the NRQL query to narrow down results by:\n- [ ] Click on the ✏️ icon inthe top right-hand corner of the dashboard\n- [ ] Click on the `Select Mobile App` variable dropdown and then click **Edit**\n- [ ] Adjust the NRQL query to narrow down results e.g. `WHERE osName = 'iOS'`"
```
Comment (seemantk) [inline dashboards/mobile/mobile.json:22]: Please change: "Click on the ✏️ icon i nthe top" -> "Click on the ✏️ icon in the top" which appears twice here

**Review by seemantk — COMMENTED** [review]
@rudouglas this looks good.  Only a couple of very minor typos to fix and this is good to go

File: dashboards/mobile/mobile.json (hunk 1 above)
Comment (rudouglas) [reply dashboards/mobile/mobile.json]: thank you kindly

**Review by seemantk — APPROVED** [review]
LGTM, thank you @rudouglas

**Pull request merged**